sys/include/net/gnrc/gnrc_netdev2.h

```diff
@@ -37,26 +37,27 @@
 extern "C" {
 #endif
 
+/**
+ * @brief   Type for @ref msg_t if device fired an event
+ */
 #define NETDEV2_MSG_TYPE_EVENT 0x1234
 
-typedef struct gnrc_netdev2 gnrc_netdev2_t;
-
 /**
- * @brief Structure holding gnrc netdev2 adapter state
+ * @brief Structure holding GNRC netdev2 adapter state
  *
  * This structure is supposed to hold any state parameters needed
- * to use a netdev2 device from gnrc.
+ * to use a netdev2 device from GNRC.
  *
  * It can be extended
  */
-struct gnrc_netdev2 {
+typedef struct gnrc_netdev2 {
```

Comment:
It's documentation related, since doxygen was complaining about the undocumented

```C
typedef struct gnrc_netdev2 gnrc_netdev_t;
```

line. As shown it is not necessary to do this, however.

This is why I put it in a separate commit.